tests/spread/rockcraft/plugin-maven/rockcraft.yaml

```diff
@@ -0,0 +1,32 @@
+name: plugin-maven
```

Comment:
Thanks for the test. I'm trying it out here with Rockcraft 1.5.3 and this project builds fine without the changes from this PR. Do you know what's going on? Ideally we'd test a case that can only work with this fix.

Reply:
See https://github.com/canonical/rockcraft/pull/728#issuecomment-2428209279